### PR TITLE
Attributes made optional in class MarketMetricInfo(TastytradeJsonDataclass)

### LINKED ISSUE #76: class MarketMetricInfo(TastytradeJsonDataclass): has faulty attributes - part 2
The following attributes are not in the get_metrics method.
I suspect it became dependent on the asset class. This was retrieved for equity options.

```
  File "tastytrade/metrics.py", line 110, in <listcomp>
    return [MarketMetricInfo(**entry) for entry in data]
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 8 validation errors for MarketMetricInfo
liquidity-value
  field required (type=value_error.missing)
liquidity-rank
  field required (type=value_error.missing)
liquidity-running-state
  field required (type=value_error.missing)
beta-updated-at
  field required (type=value_error.missing)
dividend-yield
  field required (type=value_error.missing)
listed-market
  field required (type=value_error.missing)
lendability
  field required (type=value_error.missing)
borrow-rate
  field required (type=value_error.missing)
python-BaseException
```

### PR DESCRIPTION
**Pull request by Quenos**
## Related issue(s)
Fixes #76

## Pre-merge checklist
- [x] Passing tests
- [ ] New tests added (if applicable)

### CODE REVIEW

File: tastytrade/metrics.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from datetime import date, datetime
 from decimal import Decimal
-from typing import Any, Dict, List, Optional
```
Comment (Graeme22) [inline tastytrade/metrics.py:3]: This is important for supporting earlier versions of Python

File: tastytrade/metrics.py [hunk 2]
```diff
@@ -83,17 +83,13 @@ class MarketMetricInfo(TastytradeJsonDataclass):
     iv_hv_30_day_difference: Decimal
     price_earnings_ratio: Decimal
     earnings_per_share: Decimal
-    created_at: Optional[datetime] = None
```
Comment (Graeme22) [inline tastytrade/metrics.py:86]: This does still exist so it shouldn't be removed

**Review by Graeme22 — CHANGES_REQUESTED** [review]
Hi, I appreciate the changes but there's a lot of unnecessary things and style modifications. I won't be able to accept this unless it's passing the tests and matches the rest of the project.

File: tastytrade/metrics.py [hunk 3]
```diff
@@ -161,21 +153,3 @@ def get_earnings(
     data = response.json()['data']['items']
 
     return [EarningsInfo(**entry) for entry in data]
-
-
-def get_risk_free_rate(session: Session) -> Decimal:
```
[the remark this answers is not in the document]
Comment (Quenos) [reply tastytrade/metrics.py:166]: Oops, I think I integrated from an older version.
Will do the intended changes on the current version.

**Comment by Graeme22** [comment]
> After only making changes to the attributes that failed, I still get these checks to fail after the commit, at places I didn't even touch.

Looks like you removed a blank line, there should be two after each class definition.

**Comment by Quenos** [comment]
All green, yeah :)

**Review by Graeme22 — APPROVED** [review]

**Comment by Mytak** [comment]
I get:

File "/home/dimitar/.local/lib/python3.10/site-packages/tastytrade/metrics.py", line 114, in get_market_metrics
    return [MarketMetricInfo(**entry) for entry in data]
  File "/home/dimitar/.local/lib/python3.10/site-packages/tastytrade/metrics.py", line 114, in <listcomp>
    return [MarketMetricInfo(**entry) for entry in data]
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for MarketMetricInfo
dividend-rate-per-share
  field required (type=value_error.missing)

for Ticker META.

Could you make "dividend-rate-per-share" optional too?

**Comment by Graeme22** [comment]
Would you like to open a separate PR? @Mytak

**Pull request merged**